### PR TITLE
Decrease network log severity

### PR DESCRIPTION
**Pull request by badb**
Base on these issues: https://github.com/golemfactory/golem/issues/1659 & https://github.com/golemfactory/golem/issues/1640. Log severity in these cases should be lower.

### CODE REVIEW

File: golem/network/transport/tcpnetwork.py [hunk 1]
```diff
@@ -324,8 +324,7 @@ def send_message(self, msg):
         :return bool: return True if message has been send, False otherwise
         """
         if not self.opened:
-            logger.error(msg)
-            logger.error("Send message failed - connection closed.")
+            logger.warning("Send messag %s failed - connection closed", msg)
```
Comment (jiivan) [inline golem/network/transport/tcpnetwork.py]: messag**e**

File: golem/task/tasksession.py [hunk 2]
```diff
@@ -655,8 +655,8 @@ def on_success(extracted_pkg, *args, **kwargs):
             self.result_received(extra_data, decrypt=False)
 
         def on_error(exc, *args, **kwargs):
-            logger.error("Task result error: {} ({})"
-                         .format(subtask_id, exc or "unspecified"))
+            logger.warning("Task result error: {} ({})"
+                           .format(subtask_id, exc or "unspecified"))
```
Comment (jiivan) [inline golem/task/tasksession.py]: Logging formatting

**Review by jiivan — COMMENTED** [review]
Small issues, will accept when addressed

File: golem/network/transport/tcpnetwork.py (hunk 1 above)
Comment (mfranciszkiewicz) [inline golem/network/transport/tcpnetwork.py]: Missing 'e' in message

File: golem/resource/hyperdrive/resourcesmanager.py [hunk 3]
```diff
@@ -278,7 +278,7 @@ def success_wrapper(response, **_):
             success(entry, files, task_id)
 
         def error_wrapper(exception, **_):
-            logger.error("Resource manager: error downloading %s (%s): %s",
+            logger.warning("Resource manager: error downloading %s (%s): %s",
```
Comment (mfranciszkiewicz) [inline golem/resource/hyperdrive/resourcesmanager.py]: Indent on the next line

**Review by mfranciszkiewicz — COMMENTED** [review]
LGTM, just few typos.

File: golem/resource/hyperdrive/resourcesmanager.py (hunk 3 above)
Comment (jiivan) [reply golem/resource/hyperdrive/resourcesmanager.py]: yeah, the pitfalls of ascii art indenting a.k.a. aligning with opening delimiter :)

**Review by mfranciszkiewicz — APPROVED** [review]

**Review by jiivan — APPROVED** [review]

**Pull request merged**